var/spack/repos/builtin/packages/dos2unix/package.py

```diff
@@ -19,6 +19,10 @@ class Dos2unix(MakefilePackage):
 
     depends_on('gettext')
 
+    tags = ['build-tools']
+
```

Comment:
Let's remove the tag, I don't think people want to find this package by default when running `spack external find` without argument.

Reply:
OK, thanks for the advise. I remove the tag.